core/src/main/java/tech/pegasys/web3signer/core/multikey/metadata/parser/YamlSignerParser.java

```diff
@@ -26,13 +26,16 @@
 
 import com.fasterxml.jackson.core.JsonParseException;
 import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.dataformat.yaml.YAMLGenerator;
 import com.fasterxml.jackson.dataformat.yaml.YAMLMapper;
 
 public class YamlSignerParser implements SignerParser {
 
   public static final YAMLMapper YAML_MAPPER =
       YAMLMapper.builder()
           .addModule(new SigningMetadataModule())
+          .disable(YAMLGenerator.Feature.WRITE_DOC_START_MARKER)
```

Comment:
Tests don't break when I comment this out

Reply:
Think it's still cleaner if we don't have this in the file we generate as our docs specify the format without the yaml doc start marker. I think the yaml parser handles it fine though.

Reply:
Probably because for reading the yaml file it really doesn't matter as long as its a valid yaml file. Having a doc start marker should still work, its just that it does not match our examples. Hence when writing the yaml files, we ought to not write it.